### ISSUE TITLE
Mate-pair values incorrect in meanAndStdevByPrefix.sj.txt

### ISSUE THREAD

**Issue by TimothyStephens** [alekseyzimin/masurca#30]
Hi,

The values in the meanAndStdevByPrefix.sj.txt file are different from what I provided in the config.txt file.
All of my mate-pair libraries are given a mean of 500 and a standard deviation of 100 in the meanAndStdevByPrefix.sj.txt file. Is this an expected behavior? 
The values I provided for my paired-end data are correctly displayed in meanAndStdevByPrefix.pe.txt.

**Comment by nm100** [comment]
Yeah, it is by default the values in the meanAndStdevByPrefix.sj.txt file will be 500 and 100 for mate-pair.
